homework/content.js

```diff
@@ -1,37 +1,44 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
+	var NoteContentManager = function() {
+		this._wrapper = document.querySelector('#note-content-wrapper');
+	};
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+	NoteContentManager.prototype = {
+		start() {
+			window.addEventListener('note-open', function(event) {
+			  var note = event.detail;
+			  this.resetWrapper()
+			  .then((function(){
```

Comment:
In fact, you can use Arrow function instead of binding every thing.
See:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions